### ISSUE TITLE
 insecticide has no item displays.

### ISSUE THREAD

**Issue by Hevia** [TeamMoonstorm/Starstorm2#527]

**Comment by swuff-star** [comment]
is this on its own worth an issue? ideally it gets fixed but also i think a good chunk of items are missing them. ideally could use a consolidated list down the lane rather than individual issues, though would have to double check which do/don't have displays

**Comment by Hevia** [comment]
yes this is worth an issue otherwise we will forget. We're cooking up like 30 things rn bro we dont got the cognitive load to track it all in our minds
